data/mods/gen5/random-teams.ts

```diff
@@ -147,7 +147,9 @@ export class RandomGen5Teams extends RandomGen6Teams {
 				hasMove['lightscreen'] || hasMove['reflect']
 			)};
 		case 'uturn':
-			return {cull: counter.setupType || !!counter.speedsetup || hasMove['batonpass']};
+			// Infernape doesn't want Expert Belt
```

Comment:
If this hardcode is about Expert Belt, not about U-turn's usefulness on Infernape, I'd rather hardcode it in item generation.

Reply:
infernape also doesn't want Life Orb U-turn, so the best way to prevent expert belt would be to prevent u-turn from generating in places that would otherwise give it Life Orb

Reply:
Why not add `species.id !== 'infernape'` to the `if (counter.damagingMoves.length >= 4 && ability !== 'Sturdy'))` check for ExpertBelt/LifeOrb? Or make the item that Infernape _does_ want higher priority?

Reply:
because then it'd get mixed u-turn, which it doesn't want

Reply:
Updated description to be more accurate to move issues

Reply:
Okay, that makes a lot more sense!